src/lib/ecl/attitude_fw/ecl_pitch_controller.cpp

```diff
@@ -96,15 +96,21 @@ float ECL_PitchController::control_attitude(const struct ECL_ControlData &ctl_da
 
 	/* input conditioning */
 	float airspeed = constrain_airspeed(ctl_data.airspeed, ctl_data.airspeed_min, ctl_data.airspeed_max);
-	/* calculate the offset in the rate resulting from rolling  */
-	//xxx needs explanation and conversion to body angular rates or should be removed
-	float turn_offset = fabsf((CONSTANTS_ONE_G / airspeed) *
-				  tanf(roll) * sinf(roll)) * _roll_ff;
+
+	/* Calculate desired y-axis angular rate needed to compensate for roll angle.
+	   For reference see Automatic Control of Aircraft and Missiles by John H. Blakelock, pg. 175
+	   Availible on google books 8/11/2015: 
+	   https://books.google.com/books?id=ubcczZUDCsMC&pg=PA175#v=onepage&q&f=false*/
+	float body_fixed_turn_offset = (fabsf((CONSTANTS_ONE_G / airspeed) *
+				  		tanf(roll) * sinf(roll)));
```

Comment:
Correct, this is the same compensation above without the feed forward term though you will notice that it then gets transformed from a body fixed offset (e.g angular velocity) to an euler rate offset later.  I'll be looking into the affect of the lack of feed forward this week, though theoretically it should not be applied here.  _ptich_ff would of course still drastically affect the performance, as this is only adding in compensation pitch when  the vehicle is rolling.